src/services/auth/auth.middleware.ts

```diff
@@ -43,7 +43,7 @@ export class AuthMiddleware implements NestMiddleware {
       return;
     }
 
-    req.body = verificationResult.payload;
+    req.body = verificationResult.payload as DagJWS;
```

Comment:
This is not a `DagJWS`. If the type checker is giving you issues, you can safely do `as any` since this requires further runtime schema validation to be safe to use. All that is asserted at this point is that it follows the `authSchema`.
```suggestion
    req.body = verificationResult.payload as any;
```